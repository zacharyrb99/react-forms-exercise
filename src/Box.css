.Box{
    margin: 15px;
    display: inline-block;
}