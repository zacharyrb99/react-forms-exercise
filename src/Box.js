import './Box.css'

const Box = ({id, width=50, height=50, color='red', handleRemove}) => {
    return (
        <div className='Box'>
            <div 
                className='Box' 
                style={{
                    border: '1px solid black',
                    width: `${width}px`,
                    height: `${height}px`,
                    backgroundColor: color
                }}
            >
                <button onClick={() => handleRemove(id)}> X </button>
            </div>
        </div>
    )
}

export default Box;