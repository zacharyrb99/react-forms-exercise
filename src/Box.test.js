import {render} from '@testing-library/react';
import Box from './Box';

test('Box renders without crashing', () => {
    render(<Box/>);
});

test('Box matches snapshot', () => {
    const {asFragment} = render(<Box/>);
    expect(asFragment).toMatchSnapshot();
});