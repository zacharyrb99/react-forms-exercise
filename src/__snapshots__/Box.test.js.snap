// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box matches snapshot 1`] = `[Function]`;
